projects/ngx-d3timeline/src/lib/ngx-d3timeline.component.ts

```diff
@@ -1,13 +1,20 @@
 import { Component, Input } from '@angular/core';
 import { TimelineEvent } from './timeline-event';
+import { View } from './view';
 
 @Component({
   selector: 'ngx-d3timeline',
   template: `
-    <svg></svg>
+    <svg [attr.width]="view.width" [attr.height]="view.height"></svg>
   `,
   styles: []
 })
 export class NgxD3timelineComponent {
   @Input() data: TimelineEvent[];
+
+  view: View = {
```

Comment:
Is this later changed to an input, or read from options?

Reply:
not sure maybe input, but possibly only width and height